test-script/gen_commands.py

```diff
@@ -0,0 +1,34 @@
+import glob
+import os
+import argparse
+
+modes = ['atomicity-violation', 'observability-gap', 'time-travel']
+mode_map = {
+    'atomicity-violation': 'atom-vio',
+    'observability-gap': 'obs-gap',
+    'time-travel': 'time-travel'
+}
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generate testcase commands into a file.")
+    parser.add_argument('-d', dest='docker', help="Docker account", default='ghcr.io/sieve-project/action')
```

Comment:
Just a reminder: the CI image building interval is daily (which is enough I think), but you need to add build cmds to CI file if a new operator is ported (e.g. yugabyte-operator). https://github.com/sieve-project/sieve/blob/main/.github/workflows/sieve-test.yml#L13